code/modules/events/ion_storm.dm

```diff
@@ -566,6 +566,35 @@
 							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
 
 	return message
+	
+
+/proc/corp_name()

```

Comment:
Maybe that was your input and I got you mixed up,  but someone was quite insistent that my proc end up in this file. Where do the cargo stock names come from?